PROJECT_STRUCTURE.md

```
@@ -1,79 +1,114 @@
-# 📁 Project Structure — IndicRAG v2.0
+# 📁 Project Structure — IndicRAG v2.4
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)